scarb/Cargo.toml

```diff
@@ -19,6 +19,7 @@ async-trait.workspace = true
 cairo-lang-compiler.workspace = true
 cairo-lang-defs.workspace = true
 cairo-lang-diagnostics.workspace = true
+cairo-lang-executable = "2.9.1"
```

Comment:
#1818

Reply:
If we merge #1818, this will be patched as well.